src/test/java/com/telefonica/iot/cygnus/sinks/OrionSTHSinkTest.java

```diff
@@ -68,34 +69,79 @@
     private final String attrValue = "112.9";
     private final String attrMd = "[]";
     private final String serviceHeader = "vehicles";
-    private final String servicePathHeader = "4wheels";
-    private final String destination = "4wheelsCars";
+    private final String singleServicePathHeader = "4wheels";
+    private final String multipleServicePathHeader = "4wheels,4wheels";
```

Comment:
Actually... maybe better than modify existing test to change from mono-SP to multi-SP is to keep the mono-SP tests are they are now and create new tests for the multi-SP cases. That way, you are covering both cases separately.